packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -638,16 +662,16 @@ export class CommonWell {
    * Get Patient's Network Links.
    * See: https://specification.commonwellalliance.org/services/record-locator-service/protocol-operations-record-locator-service#8771-retrieving-network-links
    *
-   * @param meta    Metadata about the request.
-   * @param id      Patient for which to get the network links.
+   * @param meta        Metadata about the request.
+   * @param patientId   Patient for which to get the network links.
    * @returns
    */
-  async getPatientsLinks(meta: RequestMetadata, id: string): Promise<PatientNetworkLinkResp> {
```

Comment:
Not keeping this as `deprecated` because the name is too close with the new one, `getPatientLinks`